core/src/main/java/com/linecorp/armeria/server/VirtualHost.java

```diff
@@ -71,6 +73,14 @@
     private final Router<ServiceConfig> router;
     private final MediaTypeSet producibleMediaTypes;
 
+    /**
+     * If accessLogger is {@code null}, it is initialized later
+     * by {@link ServerBuilder} via {@link #accessLogger(Logger)}.
+     * If it is not {@code null}, it is initialized later by {@link VirtualHostBuilder} via the constructor.
```

Comment:
I think we can remove this sentence. If it's not null, it's already initialized, right?